services/core-web/src/components/Forms/DocumentCategoryForm.tsx

```diff
@@ -56,11 +58,18 @@ export const DocumentCategoryForm: FC<DocumentCategoryFormProps> = ({
     );
   };
 
+  useEffect(() => {
+    console.log("isAmendment", isAmendment);
+  }, [isAmendment]);
+
```

Comment:
oops?

Reply:
No...I totally wanted to keep that there...